libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -54,8 +54,15 @@ + (UIImage *)imageNamed:(NSString *)name
 {
     NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    UIImage *image = [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
+    if (image) {
+        return image;
+    }
     
-    return [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
```

Comment:
@khawkins, before this change, SFSDKResourceUtils is not supporting loading image from main bundle